netsim/extra/evpn.es/frr.j2

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+{### STILL NOT WORKING - to be better investigated ###}
+
+# ESI-LAG config in FRR
+cat >/tmp/esi_lag_config <<CONFIG
+evpn mh startup-delay 1
+!
+{% for intf in interfaces if intf.type == 'lag' and intf.evpn._esi is defined %}
+interface {{ intf.ifname }}
+  evpn mh es-id {{intf.evpn._esi.id}}
```

Comment:
leaf01(config)# interface bond1
leaf01(config-if)# evpn mh es-df-pref 50000
leaf01(config-if)# evpn mh es-id 1
leaf01(config-if)# evpn mh es-sys-mac 44:38:39:FF:00:AA
leaf01(config-if)# exit

Source: NVidia Cumulus EVPN MH guide

Reply:
`evpn mh es-df-pref` is optional. as you can see, the rest is identical.
control plane messages are exchanged correcly, two frr istances see each other, DF is elected.

Reply:
You can configure
```
evpn mh uplink
```
on uplinks. Docs suggest to enable ARP/ND suppression on VXLAN side